finalPrep/annotations.js

```diff
@@ -39,4 +39,5 @@ function fizzBuzz(num1, num2, range) {
 }
 
 fizzBuzz(3, 5, 100);
-fizzbuzz(5, 8, 400);
\ No newline at end of file
+fizzbuzz(5, 8, 400);
+//Didnt get to this part of final prep.
```

Comment:
I think this is an important step to complete for the pre-work, and strongly encourage you to revisit this section on annotations.